website/azurerm.erb

```diff
@@ -282,6 +282,10 @@
                     <a href="/docs/providers/azurerm/d/policy_definition.html">azurerm_policy_definition</a>
                 </li>
 
+                <li>
+                    <a href="/docs/d/postgresql_server.html.markdown">azurerm_postgresql_server</a>
```

Comment:
the markdown file gets transformed to HTML during the Middleman compilation process - as such this can become:

```suggestion
                    <a href="/docs/d/postgresql_server.html">azurerm_postgresql_server</a>
```